### PR TITLE
Add new: Baidu Cloud Manager 2.0.0

### PR DESCRIPTION
**Pull request by zshbleaker**
- [ ] Named the cask according to the [token reference].

Because the app token uses the Chinese word "Guanjia", which means "Manager" in English.

Baidu Cloud is a leading Dropbox-like service in China.

### CODE REVIEW

**Comment by miccal** [comment]
The `sha256` needs to be set - you can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by zshbleaker** [comment]
Added.

**Comment by fanquake** [comment]
Is this different from https://github.com/caskroom/homebrew-cask/blob/master/Casks/baiducloud.rb ?

File: baiducloud-manager.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'baiducloud-manager' do
+  version '2.0.0'
+  sha256 '06a3477139a2c96fc8ceaa3a59c1a65dc0b57ec9a069f2f6356067778ae2014f'
+
+  url 'http://issuecdn.baidupcs.com/issue/netdisk/MACguanjia/BaiduNetdisk_mac_2.0.0.dmg'
```
Comment (fanquake) [inline baiducloud-manager.rb]: Use #{version} in the url stanza

File: baiducloud-manager.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'baiducloud-manager' do
+  version '2.0.0'
+  sha256 '06a3477139a2c96fc8ceaa3a59c1a65dc0b57ec9a069f2f6356067778ae2014f'
+
+  url 'http://issuecdn.baidupcs.com/issue/netdisk/MACguanjia/BaiduNetdisk_mac_#{version}.dmg'
```
Comment (jawshooah) [inline baiducloud-manager.rb]: Interpolation only works with double quotes.

File: baiducloud-manager.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+cask 'baiducloud-manager' do
```
Comment (jawshooah) [inline baiducloud-manager.rb]: Per the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md), this needs to be called `baidunetdisk`, and the file must be renamed accordingly.

File: baiducloud-manager.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  version '2.0.0'
+  sha256 '06a3477139a2c96fc8ceaa3a59c1a65dc0b57ec9a069f2f6356067778ae2014f'
+
+  url 'http://issuecdn.baidupcs.com/issue/netdisk/MACguanjia/BaiduNetdisk_mac_#{version}.dmg'
+  name '百度网盘'
```
Comment (jawshooah) [inline baiducloud-manager.rb]: Please add the English name as well, as a separate `name` stanza below this one.

File: baiducloud-manager.rb (hunk 2 above)
Comment (jawshooah) [inline baiducloud-manager.rb]: This needs a [comment](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment).

**Review by jawshooah — CHANGES_REQUESTED** [review]

File: baiducloud-manager.rb (hunk 3 above)
Comment (jawshooah) [inline baiducloud-manager.rb]: Additionally, you've added this file at the root of the repository. It belongs in the `Casks` directory.

**Review by jawshooah — CHANGES_REQUESTED** [review]
You still need to move `baidunetdisk.rb` to `Casks/baidunetdisk.rb`.

**Review by jawshooah — CHANGES_REQUESTED** [review]
The directory is `Casks`, not `Caskroom`.

**Review by jawshooah — APPROVED** [review]

**Pull request merged**